roles/mev_boost/molecule/default/molecule.yml

```diff
@@ -28,5 +28,16 @@ provisioner:
           storage-driver: "vfs"
         # Role specific variables
         mev_boost_container_name: mev-boost
+        # Use ethpandaops develop image for testing with config.yaml support
+        mev_boost_container_image: ethpandaops/mev-boost:develop
```

Comment:
not sure if we should do this? i m not sure what the latest image being picked is